### ISSUE TITLE
Progressive massive fibrosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6706]
**Preferred term label:**
Progressive massive fibrosis

**Synonyms**

**Definition (free text, please give PubMed ID)**

This condition is caused by slow-growing conglomeration of dust particles and collagen deposition in individuals (mostly coal workers) heavily exposed to inorganic dust. Progressive massive fibrosis manifests as masslike lesions, usually bilateral and in
the upper lobes. Background nodular opacities reflect accompanying pneumoconiosis, with or without emphysematous destruction adjacent to the massive fibrosis. Lesions similar to progressive massive fibrosis sometimes occur in other conditions, such as sarcoidosis and talcosis.

PMID:18195376

Comment
See Figure 56 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
